plugins/goosebit_simple_stats/pyproject.toml

```
@@ -0,0 +1,23 @@
+[project]
+name = "goosebit-simple-stats"
+version = "0.1.0"
+description = ""
+authors = [
+    {name = "Brett Rowan", email = "121075405+b-rowan@users.noreply.github.com"}
+]
+readme = "README.md"
+requires-python = ">=3.11,<4.0"
+dependencies = ["goosebit @ file:///home/brett/PycharmProjects/goosebit"]
```

This needs to be changed

---

Can we remove that dependency altogether? Or change it to
```suggestion
dependencies = ["goosebit"]
```
?

I would like to have the plugins installable with `pip` directly from the Git repo. The relative path won't work.
Or can we make the plugin an optional dependency like `postgresql`? That would work for me too.